source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -54,6 +71,15 @@ content: |
            HashMap<String, BsonDocument> schemaMap = new HashMap<String, BsonDocument>() {{
              put("medicalRecords.patients", BsonDocument.parse(jsonSchema));
            }}
+     .. tab::
+        :tabid: nodejs
+        
+        .. code-block:: javascript
+        
+          const dataNamespace = "medicalRecords.patients";
+          const patientSchema = {
+            [dataNamespace]: jsonSchema
+          }
```

Comment:
nit: indentation is off here, though it still renders fine

There should be 11 spaces before each line of code (to line up with the "c" in `code-block`) but here there are only 10

Reply:
Fixed